security/docker/Dockerfile.node-agent-k8s

```diff
@@ -2,13 +2,17 @@
 ARG BASE_DISTRIBUTION=default
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
+# hadolint ignore=DL3006
 FROM istionightly/base_debug as default
+# hadolint ignore=DL3005,DL3008
 RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates && update-ca-certificates && apt-get clean  && rm -rf /var/lib/apt/lists/*
```

Comment:
This is probably one place where we should pin the version. I understand why we may not want to do that in this PR though.